racetime/migrations/0001_initial.py

```diff
@@ -49,7 +49,7 @@ class Migration(migrations.Migration):
                 ('short_name', models.CharField(help_text='An abbreviation or other short identifier, e.g. "SM64".', max_length=16, unique=True)),
                 ('slug', models.CharField(help_text='A unique identifier for this category used in the URL, e.g. "tetris-99".', max_length=255, unique=True)),
                 ('image', models.ImageField(blank=True, help_text='Recommended size: 285x380. No larger than 100kb.', null=True, upload_to='')),
-                ('info', models.TextField(blank=True, help_text='Displayed on the category page. Limited use of HTML is allowed.', null=True)),
+                ('info', models.TextField(blank=True, help_text='Displayed on the category page. Limited use of HTML is allowed. Supports Markdown.', null=True)),
```

Comment:
You cannot edit a previously run migration.

Well, naturally you _can_, but it won't work. Django never re-runs a migration on the same database, so if you make a change to the schema you need to generate a new migration to cover that change.

You can do this automatically by running the `makemigrations` command, i.e. `docker-compose exec racetime.web python manage.py makemigrations`.

More info: https://docs.djangoproject.com/en/3.0/topics/migrations/

Reply:
Dealt with converting this. This is what I get for "control f" "copy" "paste"